crates/bevy_ecs/src/schedule/executor/simple.rs

```diff
@@ -98,6 +98,8 @@ impl SystemExecutor for SimpleExecutor {
 }
 
 impl SimpleExecutor {
+    /// Creates a new simple executor for use in a [`Schedule`](super::super::Schedule).
+    /// This calls each system in order and immediately calls [`System::apply_deferred`](crate::system::System::apply_deferred).
```

Comment:
james's comment wasn't addressed. This could be improved by pulling the links into footnotes.